code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -447,7 +447,6 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 		if (burn_rate)
 			user << "<span class='notice'>[src] is already packed.</span>"
 			return
-		burn_rate = 0.1

```

Comment:
This is not going to cause more problems. In attack_self() pipes check for a positive burn_rate value, and if they find one they unpack the pipe, setting it to 0. If the pipe is packed (aka a positive burn rate) you cannot pack it as above. This change will lead to both:
A) A burn_rate that is always 0
B) An infinitely packable pipe.

Reply:
fixed